Author: Pino Toscano <pino@debian.org>
Description: nfs: provide a fallback PATH_MAX definition
 Define PATH_MAX to 4096 if not defined already -- this way kio_nfs can build
 again on Hurd.
Forwarded: not-needed
Last-Update: 2016-06-18

--- a/kde-runtime-16.04.2/kioslave/nfs/nfsv3.cpp
+++ b/kde-runtime-16.04.2/kioslave/nfs/nfsv3.cpp
@@ -59,6 +59,9 @@
 #define NFSVERS 3UL
 
 #define NFS3_MAXDATA    32768
+#ifndef PATH_MAX
+# define PATH_MAX 4096
+#endif
 #define NFS3_MAXPATHLEN PATH_MAX
 
 NFSProtocolV3::NFSProtocolV3(NFSSlave* slave)
